docs/guides/api-gateway/kubernetes.mdx

```diff
@@ -1,430 +1,558 @@
-  also designed for dynamic infrastructure provisioning and advanced traffic routing.
-- The Gateway API is role-oriented, allowing DevOps/platform/infrastructure
-  engineering teams to control clusters, policies, and permissions centrally,
-  while still allowing API developers to self-service endpoints to their services
-  with route objects.
-- The Gateway API provides a common interface to K8s networking, removing
-  reliance on vendor-specific configurations.
-- Many developers can integrate their APIs or apps onto a single `GatewayClass`
-  or `Gateway`, using `HTTPRoute`s, without affecting others.
-
-For the scope of this guide, and based on this recommendation, we'll focus on
-Gateway API.
-
-## **Step 0**. Deploy the example ngrok API service
-
-If you're building a proof of concept with ngrok as the API gateway for your
-APIs in Kubernetes, you can use our [example ngrok API
-service](https://github.com/ngrok-samples/ngrok-api-service). This service
-responds to all routes with information about the connection&mdash;think of it
-as the `pong` to your `curl`.
-
-:::note
-If you already have an API service, you can skip this step&mdash;just be aware
-that you'll need to edit the example configurations below to properly configure
-the ngrok Kubernetes Operator.
-:::
-
-To deploy the ngrok API service to your cluster, apply the [`deployment.yaml`
-file](https://github.com/ngrok-samples/ngrok-api-service/blob/main/kubernetes/deployment.yaml)
-to your cluster, which first creates a new `prod` namespace if one doesn't
-already exist.
-
-```bash
-kubectl apply -f https://raw.githubusercontent.com/ngrok-samples/ngrok-api-service/refs/heads/main/kubernetes/deployment.yaml
+- **[`kubectl`](https://kubernetes.io/docs/tasks/tools/) and [Helm](https://helm.sh/docs/intro/install/)** installed on your local workstation.
+- **A Kubernetes cluster**.
+- **A reserved domain**, which you can get [in the ngrok
+  dashboard](https://dashboard.ngrok.com/domains) or with the [ngrok
+  API](/docs/api/resources/reserved-domains/#create-reserved-domain).
+  - You can choose from an ngrok subdomain or bring your own [custom branded
+    domain](/docs/guides/other-guides/how-to-set-up-a-custom-domain), like
+    `https://api.example.com`.
+  - We'll refer to this domain as `<YOUR_NGROK_DOMAIN>`.
+- An [**Auth0
+  account**](https://auth0.com/signup?place=header&type=button&text=sign%20up)
+  (free is fine) for creating an API and generating JWTs.
+
+### Deploy demo API services
+
+If you want to quickly wire up a POC using ngrok, we recommend using our [demo
+API service](https://github.com/ngrok-samples/api-demo), which responds with details
+about your requests.
+
+You can also bring your own services to this guide—in that case, make sure to
+change service names, target ports, and namespaces accordingly in the resources
+you'll define in the next few steps.
+
+The two manifests below provision pods and resources for `abc` on
```

Comment:
This sentence is confusing to me. suggested rephrasing:

"The following manifests create two different but similar deployments that we will direct traffic to. Using the container environment variables section, they are configured slightly differently to demonstrate that requests are actually being routed to different destinations."